src/entities/viewer/model/store.ts

```diff
@@ -1,19 +1,42 @@
 import { sample, createEvent, createStore, createEffect } from 'effector';
 import { User } from './types';
-import { getUser, login, logout, getAuthState } from './api';
+import {
+  getUser,
+  login,
+  logout,
+  getAuthState,
+  accessToken,
+  updateUser,
+  deleteUser,
+  getApiToken,
+} from './api';
 
 export const loginRequested = createEvent();
 export const logoutRequested = createEvent();
+export const userChanged = createEvent<Partial<User>>();
 
 export const loginFx = createEffect(login);
 export const logoutFx = createEffect(logout);
 export const getUserFx = createEffect(getUser);
+export const getTokenFx = createEffect(accessToken);
```

Comment:
change naming, you have 2 tokens here, which is which? userToken and managementToken or what kind of tokens are they?

Reply:
this one is accessToken I guess

Reply:
fixed